CHANGELOG.md

```diff
@@ -47,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Improved the failed hierarchy migration logs. The logs now contain more specific reasons for the failure and the frequency of repeated logs is suppressed.
 - SpatialWorldSettings is now the default world settings in supported engine versions.
 - Worker SDK version compatibility is checked at compile time.
+- Unreal GDK now uses SpatialOS 15.0.0-preview-2.
```

Comment:
Thought we moved to preview-3 or 4 now?

Reply:
baby steps

Reply:
Preview 3 has breaking changes around event tracing - will be addressed in a follow-up PR.